README.md

```diff
@@ -155,6 +155,41 @@ $http = new Server($socket, function (ServerRequestInterface $request) {
 });
 ```
 
+The `getServerParams(): mixed[]` method can be used to
+get server-side parameters similar to the `$_SERVER` variable.
+The following parameters are currently available:
+
+* `remote_address`
+  The IP address of the request sender
+* `remote_port`
+  Port of the request sender
+* `server_address`
+  The IP address of the server
+* `server_port`
+  The port of the server
+* `request_time`
+  Unix timestamp when the complete request header has been received,
+  as integer similar to `time()`
+* `request_time_float`
+  Unix timestamp when the complete request header has been received,
+  as float similar to `microtime(true)`
+* `https`
+  Set to 'on' if the request used HTTPS, otherwise it will be set to `null`
```

Comment:
AFAICT `$_SERVER['https']` will be unset for plain HTTP, does it really make sense to assume a `null` value here?